### PR TITLE
Improve Markdown rendering of README.md

### LINKED ISSUE #3: README.md formatting doesn't render well on GitHub
There are many sections in `README.md` which look good in plaintext but are not rendered nicely on the front page of the GitHub repository. Some of those sections might be better formatted as a Markdown table, and others as monospace text.

### PR DESCRIPTION
**Pull request by akaihola**
Fixes #3.

- force line breaks in contact information and some paragraphs
- MathJax equations
- name frequencies as Markdown tables
- monospace for per-country statistics

See the [the rich diff](4/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) to view the difference in rendered output.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,21 +55,23 @@ have all been independently lassified by several native speakers.
...
 "Esra". This is a boy's name in Germany, but a girl's name in Turkey.
 
 Or consider the following first names:
 
-Mitja      male Russian name
-Elizaveta  rare name; looks like misspelled "Elizabeta"
-Roelf      rare name; looks like German "Rolf" with an erroneous 'e'
+| name      | description                                               |
+| ----      | -----------                                               |
+| Mitja     |  male Russian name                                        |
```
Comment (Mo-Gul) [inline README.md]:
```suggestion
| Mitja     | male Russian name                                        |
```

File: README.md [hunk 2]
```diff
@@ -83,443 +84,446 @@ Thus, "Jun+Wei" represents the names "Jun-Wei", "Jun Wei" and "Junwei".
...
-Names from Turkey         :  1806
-Names from Arabia/Persia  :  2025
-Names from Israel         :  1067
-Names from China          :  7334
-Names from India/Sri Lanka:  1455
-Names from Japan          :  1384
-Names from Korea          :  1376
-Names from Vietnam        :   307
+| category              |  number |
+| --------              |  ------ |
```
Comment (Mo-Gul) [inline README.md]:
```suggestion
| --------              |  ------:|
```

File: README.md [hunk 3]
```diff
@@ -83,443 +84,446 @@ Thus, "Jun+Wei" represents the names "Jun-Wei", "Jun Wei" and "Junwei".
...
+| Number of first names |   44568 |
+| – girl's names        |   16921 |
+| – boy's  names        |   17740 |
+| – unisex names        |    9907 |
+| Equivalent name pairs |     734 |
+
+| area                                   | number |
+| ----                                   | ------ |
```
Comment (Mo-Gul) [inline README.md]:
```suggestion
| ----                                   | ------:|
```

File: README.md [hunk 4]
```diff
@@ -83,443 +84,446 @@ Thus, "Jun+Wei" represents the names "Jun-Wei", "Jun Wei" and "Junwei".
...
+| Names from U.S.A.                      |  3916  |
+| Names from Italy                       |  2871  |
+| Names from Malta                       |   526  |
+| Names from Portugal                    |  1041  |
+| Names from Spain                       |  1645  |
+| Names from France                      |  1821  |
+| Names from Belgium                     |  1518  |
+| Names from Luxembourg                  |   564  |
+| Names from the Netherlands             |  3302  |
+| Names from East Frisia                 |  2589 ("Ostfriesland") |
```
Comment (Mo-Gul) [inline README.md]: so the alignment looks nice, I suggest to move the bracket term to a third column.

Comment (akaihola) [reply README.md]: I made a third column.

**Pull request merged**